ui/raidboss/data/triggers/o7s.js

```diff
@@ -110,6 +110,7 @@
       id: 'O7S Orb Marker',
       regex: /1B:........:(\y{Name}):....:....:0017:0000:0000:0000:/,
       condition: function(data, matches) { return matches[1] == data.me; },
+      //this is fine in German. comment just for confirmation and can be deleted
```

Comment:
Do you think it would be clearer to just return an object that said like {en: 'Orb Marker', de: 'Orb Marker'} even if they were the same, just to be obvious that the translation was complete?

Feel free to leave these sort of transitory comments to me as code review by just clicking on the line and leaving it (like I'm leaving this comment).  At the very least, go ahead and remove this comment in code before I merge this.

Reply:
Ok! Could you remove this comment here and do that in this patch?

Reply:
The comment should already be removed and the new stuff pushed o.o

Reply:
It still looks like it's in the patch.  If I look at 758099 then it's only changes to the byakko file.  If this is changed for you locally, can you make sure it's committed and pushed?